content/rust/package/publish-deb-release-multi-arch.md

```diff
@@ -0,0 +1,468 @@
+---
+date: 2000-01-01T00:00:00+00:00
+title: Publish multi-Architecture Debian packages for Rust
+author: KN4CK3R
+draft: true
+image: /static/header-images/rust-deb.png
```

Comment:
this can be removed (and the header-images) because we have an illustrator creating images for each tutorial  :)

Reply:
I was hoping that you don't need my paint skills. :wink: